packages/web/src/components/spa/HackAUBG/JourneySection/JourneyMobile/journey_mobile_old.css

```diff
@@ -0,0 +1,34 @@
+.journey-mobile-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+.journey-mobile-content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+.journey-mobile-title {
```

Comment:
Add newline between classes

Reply:
this is about the old css that has been removed